src/data/nat/modeq.lean

```diff
@@ -28,9 +28,14 @@ namespace nat
 /-- Modular equality. `n.modeq a b`, or `a ≡ b [MOD n]`, means that `a - b` is a multiple of `n`. -/
 @[derive decidable]
 def modeq (n a b : ℕ) := a % n = b % n
-
 notation a ` ≡ `:50 b ` [MOD `:50 n `]`:0 := modeq n a b
 
+/-- Invalid modular congruence. `n.modneq a b`, or `a ≢ b [MOD n]`,
+means that `a - b` is NOT a multiple of `n`. -/
+@[reducible] def modneq (n a b : ℕ) := ¬ (a ≡ b [MOD n])
+notation a ` ≢ `:50 b ` [MOD `:50 n `]`:0 := modneq n a b
+lemma modneq_iff (n a b : ℕ):  a ≢ b [MOD n] ↔ a % n ≠ b % n := by refl
```

Comment:
Same suggestions as for ZMOD. Just pointing this line out in particular to make sure the spaces around the colon are corrected.
```suggestion
lemma modneq_iff (n a b : ℕ) : a ≢ b [MOD n] ↔ a % n ≠ b % n := by refl
```

Reply:
https://github.com/leanprover-community/mathlib/pull/14682/commits/8c3bdf8899c83650c9ef528859a5eb11e29d0d25